hack/lib/strimzi.bash

```diff
@@ -53,6 +53,7 @@ function install_strimzi_cluster {
           transaction.state.log.min.isr: 2
           inter.broker.protocol.version: "3.2"
           auto.create.topics.enable: "false"
+          min.insync.replicas: 2
```

Comment:
I think I tried this before, and there were failures...

Reply:
Ok, I'll check the errors